src/main/java/frc/robot/commands/teleop/vision/VisionRun.java

```diff
@@ -1,60 +1,51 @@
-package frc.robot.commands.teleop;
+package frc.robot.commands.teleop.vision;
 
-import frc.robot.subsystems.Ringlight;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.Constants;
-import frc.robot.controls.Driver;
 import frc.robot.Robot;
 
-public class TeleopRinglight extends CommandBase
+public class VisionRun extends CommandBase
```

Comment:
In general, I'm just wondering if it would be easier to name the commands `StopIntake` `SpinIntake` etc so that is it less confusing for kids.

Reply:
Since we have folders now for the commands, the naming scheme it shouldn't matter